site/README.md

```diff
@@ -0,0 +1,18 @@
+# IPFS Distributions
+
+
+## Development
+
+Harpjs setup with
+
+* Live reload using browser-sync
+* JavaScript
+  * bundeling using webpack
+  * transpiling using babel
+* Linting using eslint
```

Comment:
are these required? why can't we keep these out? it makes it daunting to modify _websites_ when suddenly you have to know a ton more tools and know how to debug them when something goes wrong.

there's a lot to be said for simplicity and reducing cognitive complexity to the user.